_posts/2015-07-05-personal.markdown

```diff
@@ -10,14 +10,12 @@ thumbnail: personal.jpg
 license: MIT License
 license_link: https://github.com/PanosSakkos/personal-jekyll-theme/blob/master/LICENSE
 ---
-
```

Comment:
To ensure similar formatting for all theme records then please do not remove the empty newline here, thanks!